workflows/cloudify_system_workflows/snapshots/postgres.py

```diff
@@ -618,16 +511,32 @@ def _get_clear_tables_queries(self, preserve_defaults=False):
             self._add_preserve_defaults_queries(queries)
         return queries
 
+    def _get_roles_constraints(self, constraint='IMMEDIATE'):
+        tables = (
+            'groups_roles',
+            'groups_tenants',
+            'users_roles',
+            'users_tenants',
+        )
+        return [
+            "ALTER TABLE {0} ALTER CONSTRAINT {0}_role_id_fkey {1};".format(
+                t, constraint) for t in tables
+        ]
+
+    def _get_status_reporter_deletes(self):
+        return [
+            "DELETE FROM users_roles WHERE user_id IN ( "
+            "SELECT id FROM users WHERE username='manager_status_reporter');",
+            "DELETE FROM users WHERE username='manager_status_reporter';",
```

Comment:
also rabbitmq and db status reporter

Reply:
those https://github.com/cloudify-cosmo/cloudify-common/blob/a6fff6faf40474e64d6364d86c8af107298d4ce8/cloudify/cluster_status.py#L17-L19